README.md

```diff
@@ -28,3 +28,38 @@ Open two shell/terminal sessions:
 _Refer to `manage -h` for additional usage information._
 
 Once services are started, visit [http://localhost:8080](http://localhost:8080) to see the app running and visit [http://localhost:8050](http://localhost:8050) to see any outbound mail the app is sending (it won't actually send any email message in the development environment.)
+
+## Deploying to Openshift
+
+**Please make sure you have [Openshift Developer Tools](https://github.com/BCDevOps/openshift-developer-tools/tree/master/bin) installed and available on your path.**
+
+### Param Files
+
+Navigate to the openshift folder and view the existing param files.  Go into email-verification-agent-deploy.param and email-verification-service-deploy.param and set  
+`NAMESPACE_NAME=a99fd4`  
+and set it to whatever your project namespace is.
+
+Also navigate to settings.sh and at the line `export PROJECT_NAMESPACE=${PROJECT_NAMESPACE:-a99fd4}` replace `a99fd4` with your project namespace name
+
+### Builds
+Now that our param files are set up, we're ready to start the builds using the openshift developer script [here](https://github.com/BCDevOps/openshift-developer-tools/tree/master/bin#generate-the-build-and-images-in-the-tools-project-deploy-jenkins).     
+Now that the builds have been completed and tagged it's time to start the deployment
```

Comment:
Images have not been tagged yet, that comes later, after the deployment configurations have been published.

Reply:
changed wording to specify that images are tagged after deployment